src/test/kotlin/racingcar/game/RacingGameTest.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.game
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.tuple
+import org.junit.jupiter.api.Test
+import racingcar.car.Car
+import racingcar.car.MockEngine
+
+class RacingGameTest {
+    @Test
+    fun `게임 한 라운드가 진행 되면 차가 움직이거나 움직이지 않는다`() {
```

Comment:
```suggestion
    fun `게임 한 라운드를 진행한다`() {
```
차는 움직이거나 혹은 움직이지 않거나 둘 중 하나이므로 명시하지 않아도 될 것 같아요.